server/lib/configuration.js

```diff
@@ -274,6 +286,10 @@ var conf = module.exports = convict({
 // on the path, we'll use that, otherwise we'll name it 'ephemeral'.
 conf.set('process_type', path.basename(process.argv[1], '.js'));
 
+if (conf.get('env') == 'development') {
```

Comment:
nit: `===`

Not sure if we want to add a brief comment as well about us forcing `cspEnabled` specifically in development. For example, this comment which we removed in a previous diff above:
```js
// send CSP headers in development mode
```

---

Also... weird, why wouldn't Travis/JSHint/jscs not complain about double `==` instead of triple `===`?